supplier_api_tester/supplier_api_tester/models.py

```diff
@@ -98,3 +98,4 @@ class Booking:
     barcode_position: str
     barcode: Optional[str]
     tickets: Optional[Dict[str, List[str]]]
+    is_cancelled: Optional[bool]
```

Comment:
As we discussed there is no need to store this information. If we do the cancellation twice we will know that it will be canceled after the first approach.